test/e2e/custom_url_test.go

```diff
@@ -22,14 +22,100 @@ import (
 	consoleapi "github.com/openshift/console-operator/pkg/api"
 	routesub "github.com/openshift/console-operator/pkg/console/subresource/route"
 	"github.com/openshift/console-operator/test/e2e/framework"
-	"github.com/openshift/library-go/pkg/crypto"
 )
 
 const (
 	consoleRouteCustomTLSSecretName   = "console-route-custom-tls"
 	downloadsRouteCustomTLSSecretName = "downloads-route-custom-tls"
 )
 
+const (
+	validCertificate = `-----BEGIN CERTIFICATE-----
```

Comment:
You should also test the invalid case. I was recently reminded that I actually contributed a script to generate all kinds of certificates you may need, see https://www.github.com/kubernetes/kubernetes/blob/5479306b1326bcbf29ae2ca0694a36555ed1ce35/staging/src/k8s.io/apiserver/pkg/util/webhook/gencerts.sh